ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -35,6 +59,34 @@ export default class AlertsViewControls extends React.Component {
     setFiltersState({ framework }, this.fetchAlertSummaries);
   };
 
+  updateCurrentAlert = (currentAlert) => {
+    this.alertsRef[currentAlert].current.scrollIntoView({
+      behavior: 'smooth',
+      block: 'start',
+    });
+
+    this.setState({ currentAlert });
+  };
+
+  onScrollAlert = (type) => {
+    const { alertsLength } = this.state;
+    let { currentAlert } = this.state;
+    const scrollTypes = {
+      prev: 'prev',
+      next: 'next',
+    };
+
+    if (type === scrollTypes.next) {
+      currentAlert = currentAlert === alertsLength - 1 ? 0 : currentAlert + 1;
+    }
+
+    if (type === scrollTypes.prev) {
+      currentAlert = currentAlert <= 0 ? alertsLength - 1 : currentAlert - 1;
```

Comment:
I think the `Next` button should be disabled when you reach the last alert summary.
Same for first alert summary and `Previous` button.

Right now it goes in a circle.